crates/ef-testing/src/sequencer/constants.rs

```diff
@@ -0,0 +1,101 @@
+use std::{collections::HashMap, sync::Arc};
+
+use blockifier::block_context::BlockContext;
+use lazy_static::lazy_static;
+use starknet::core::types::{contract::legacy::LegacyContractClass, FieldElement};
+use starknet_api::{
+    block::{BlockNumber, BlockTimestamp},
+    core::{ChainId, ClassHash, ContractAddress, PatriciaKey},
+    hash::StarkFelt,
+};
+
+lazy_static! {
+    // Chain params
+    pub static ref CHAIN_ID: u64 = 1_263_227_476;
```

Comment:
I'd rather have 0x4b4b5254 literal `0x4b4b5254` so that it's clearer that it's the ascii value of some string